dashboard/admin.py

```diff
@@ -79,5 +87,44 @@ def save_model(self, request, obj, form, change):
         obj.id = canvas_id_to_incremented_id(obj.canvas_id)
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
+class LogResource(resources.ModelResource):
+    class Meta:
+        model = Log
+
+# This is a local class for logs that adds in Export and disables some actions on the admin
+class MyLALogAdmin(ExportMixin, LogAdmin):
+    resource_class = LogResource
+    # Remove adding and editing for logs
+    @staticmethod
+    def has_add_permission(request):
+        return False
+    @staticmethod
+    def has_change_permission(request):
+        return False
+    @staticmethod
+    def has_delete_permission(request):
+        return False
```

Comment:
I think the `obj=None` is needed for change and delete in order to see the item level pages. Seeing error pages without these changes. Looks like you need to recreate the method signatures here for the overrides? https://docs.djangoproject.com/en/3.2/ref/contrib/admin/
```suggestion
    def has_change_permission(request, obj=None):
        return False
    @staticmethod
    def has_delete_permission(request, obj=None):
        return False
```

Reply:
Okay, thanks! I had it in there originally and must have removed too much converting to a staticmethod.